poetry.lock

```diff
@@ -2620,7 +2620,7 @@ multidict = ">=5.0,<6.0"
 sanic-routing = ">=0.7,<1.0"
 ujson = {version = ">=1.35", markers = "sys_platform != \"win32\" and implementation_name == \"cpython\""}
 uvloop = {version = ">=0.5.3", markers = "sys_platform != \"win32\" and implementation_name == \"cpython\""}
-websockets = ">=10.0"
+websockets = ">=10.0,<11.0"
```

Comment:
To double-check, installing from this lock file in a fresh venv with python 3.10 and then running any rasa command completes successfully?

Reply:
Yes, it does. I created a venv in Python 3.10 to validate it just now as well